Rover/Rover.cpp

```diff
@@ -344,22 +344,23 @@ void Rover::ahrs_update()
  */
 void Rover::gcs_failsafe_check(void)
 {
-    if (!g.fs_gcs_enabled) {
+    if (g.fs_gcs_enabled == FS_GCS_DISABLED) {
         // gcs failsafe disabled
         return;
     }
 
-    // check for updates from GCS within 2 seconds
     const uint32_t gcs_last_seen_ms = gcs().sysid_myggcs_last_seen_time_ms();
-    bool do_failsafe = true;
     if (gcs_last_seen_ms == 0) {
-        // we've never seen the GCS, so we never failsafe for not seeing it
```

Comment:
This comment is quite a useful one and we shouldn't lose it.

Reply:
added back